doc/source/tutorials/bipartite_matching_maxflow/assets/bipartite_matching_maxflow.py

```diff
@@ -0,0 +1,51 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph(
+    9,
+    [(0, 4), (0, 5), (1, 4), (1, 6), (1, 7), (2, 5), (2, 7), (2, 8), (3, 6), (3, 7)],
+    directed=True
+)
+
+# Assign nodes 0-3 to one side, and the nodes 4-8 to the other side
+for i in range(4):
+    g.vs[i]["type"] = True
```

Comment:
`g.vs[range(4)] = True` also works; not sure if you actually want to show that or just want to be explicit about the looping, both are okay for me